cluster-autoscaler/core/autoscaler.go

```diff
@@ -66,6 +67,13 @@ func NewAutoscaler(opts AutoscalerOptions) (Autoscaler, errors.AutoscalerError)
 	if err != nil {
 		return nil, errors.ToAutoscalerError(errors.InternalError, err)
 	}
+
+	// These metrics should be published only once.
+	for _, nodeGroup := range opts.CloudProvider.NodeGroups() {
+		metrics.UpdateNodeGroupMin(nodeGroup.Id(), nodeGroup.MinSize())
+		metrics.UpdateNodeGroupMax(nodeGroup.Id(), nodeGroup.MaxSize())
+	}
```

Comment:
can we maybe include current nodeGroup size? smth like `nodeGroup.TargetSize()` WDYT ?

Reply:
adding the current size would require this metric to updated more frequently for it to be accurate. which brings us back to the cardinality issue.

Reply:
The current ready nodes can calculated from other kube metrics as well.

Reply:
agreed, ready notes metrics is not required in CA